src/app/pages/project/project.component.html

```diff
@@ -2,36 +2,77 @@
     <nb-card class="dd-card-container col-sm-12">
         <nb-card-body class="dd-card">
             <div class="dd-card-header">
-                <h3>Cyclones</h3>
+                <h3>{{ activeProject.name }}</h3>
             </div>
             <section class="dd-card-content row">
                 <div class="ratings-container col-sm-4">
+                    <div class="repo-container py-2">
+                        <select type="text" class="form-control" [(ngModel)]="activeProjectRepo" (ngModelChange)="onChange($event)">
+                            <option *ngFor="let repo of activeProject.projectRepos" [ngValue]="repo">
+                                {{ repo.name }}
+                            </option>
+                        </select>
+                        <button class="add-proj-btn" (click)="refreshMetrics()">Pull Sonar Report</button>
+                    </div>
                     <nb-list>
                         <nb-list-item>
                             Project quality rating
                             <span class="ratings status-ok">87</span>
                         </nb-list-item>
                         <nb-list-item>
                             Current overall coverage
-                            <span class="ratings status-ok">87</span>
+                            <span class="ratings status-ok">
+                                    {{activeProjectRepo.qualityReports[0].sonarQubeReport.overallCoverage.toFixed(1)}}
```

Comment:
Create a method which returns the value needed here or at least for the sonarQubeReport. In this way, you'll also be able to guard yourself from nulls and it makes the code more readable and you can also remove the duplicated line from below.